src/components/cc-tcp-redirection/cc-tcp-redirection.js

```diff
@@ -70,7 +70,7 @@ export class CcTcpRedirection extends LitElement {
   /**
    * @param {string} namespace
    * @param {boolean} isPrivate
-   * @returns {string}
+   * @returns {Node|string|void}
```

Comment:
question: isn't void useless here?

Reply:
It's here to cover the following case:

- you call this method with `private => false` a `namespace` that is neither `default` or `cleverapps`,
- this method won't return anything, hence `void`.

If you remove it, TS will say something like `not all paths return a value` as a warning.